_format/1.1/index.md

```diff
@@ -2299,8 +2300,12 @@ An error object **MAY** have the following members:
 
 * `id`: a unique identifier for this particular occurrence of the problem.
 * `links`: a [links object][links] containing the following members:
-  * `about`: a [link][links] that leads to further details about this
-    particular occurrence of the problem.
+  * `about`: a [link][link] that leads to further details about this
+    particular occurrence of the problem. When derefenced, this URI **SHOULD**
+    return a human-readable description of the error.
+  * `type`: a [link][link] or array of links that identifies the type of error 
```

Comment:
Why allow an array of links for `type` but not for other links?

Reply:
Allowing an array for `about` and other existing links would be a breaking change (probably + unfortunately), but `type` should be an array because an error can obviously be of multiple types, and there's precedent for array-shaped links with `profile`. Ideally, I'd like to have all new links going forward hold an array for consistency

Reply:
I am not sure it's a benefit to allow all links to hold an array - consider the `self` link for instance.

And there is a singular aspect to defining each error in an error object. `code` is singular as are the other links. Furthermore, `type` throughout the spec is used to define a singular type of resource, so it seems a nice parallel to define a singular error link. Having a single link to turn to for info can be a nice simplification.

Perhaps I'm missing something. I'm definitely open to hearing some use cases that make you think an array here would be desirable.

Reply:
Having discussed this a bit more with @ethanresnick, I am now ok with allowing a singular link or array of links here. The array could be used for further refinements of a `type` - from general to specific. This could allow error objects in an API to evolve while still supporting old clients looking for a particular link.